### PR TITLE
Create mr-imint.md

### PR DESCRIPTION
**Pull request by vvizard-**
Mr IMINT Author information

### CODE REVIEW

File: content/author/mr-imint.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+date: 2017-09-12T15:40:00+02:00
+title: Mr IMINT
+---
+
+_Mr IMINT_ is a small shareholder of Imint AB. He lives in Oslo, Norway, and have no current or prior connection to Imint AB.
```
Comment (simonmika) [inline content/author/mr-imint.md]: It should be: "and  _has_ no current"

Comment (simonmika) [reply content/author/mr-imint.md]: As you spend quite some time on doing Imint related research I think there you can mention your areas of interest when it comes to Imint.

**Review by simonmika — CHANGES_REQUESTED** [review]
One grammatical error.

File: content/author/mr-imint.md (hunk 1 above)
Comment (vvizard-) [reply content/author/mr-imint.md]: Ok, I'll fix the errors, and update it a bit.

**Review by simonmika — APPROVED** [review]
Great

**Pull request merged**